src/main/java/org/fundacionjala/at15/katas/pokerhands/gonzalo/Cards.java

```diff
@@ -0,0 +1,9 @@
+package org.fundacionjala.at15.katas.pokerhands.gonzalo;
+
+import java.util.List;
+
+public abstract class Cards {
+
+    public abstract List<String> cards(String suit);
```

Comment:
Always try to use a verb for method names, this way it helps to describe what the method is doing (in this case, does it get cards? does it read cards? does it create cards?)